src/matricize.jl

```diff
@@ -74,56 +74,41 @@ function matricize(a::AbstractArray, permblock1::Tuple, permblock2::Tuple)
 end
 
 # ====================================  unmatricize  =======================================
-function unmatricize(
-  m::AbstractMatrix,
-  axes::Tuple{Vararg{AbstractUnitRange}},
-  biperm::AbstractBlockPermutation{2},
-)
+function unmatricize(m, axes, biperm::AbstractBlockPermutation{2})
```

Comment:
What's the motivation for removing the `AbstractMatrix` constraint?

Reply:
Thanks. Let's put that constrain back for now to make it clearer what kind of input is expected. I'd be fine removing it in the future when we have a use case that isn't `AbstractMatrix`.